homeassistant/components/light/isy994.py

```diff
@@ -72,7 +72,9 @@ def turn_on(self, brightness=None, **kwargs) -> None:
     @property
     def state_attributes(self):
```

Comment:
https://github.com/home-assistant/home-assistant/commit/a3db0ec231f467391438bf01c8cf466f4d5ec631 was not correctly implemented.